README.md

```diff
@@ -134,11 +134,13 @@ manager.MaximumFreeLargePoolBytes = maxBufferSize * 4;
 manager.MaximumFreeSmallPoolBytes = 100 * blockSize;
 ```
 
+### Usage Guidelines
+
 While this library strives to be very general and not impose too many restraints on how you use it, its purpose is to reduce the cost of garbage collections incurred by frequent large allocations. Thus, there are some general guidelines for usage that may be useful to you:
 
 1. Set the `blockSize`, `largeBufferMultiple`, `maxBufferSize`, `MaximumFreeLargePoolBytes` and `MaximumFreeSmallPoolBytes` properties to reasonable values for your application and resource requirements.
 2. Always dispose of each stream exactly once.
-3. Never call `ToArray` and avoid calling `GetBuffer` if possible. Instead, use `GetReadOnlySequence` for reading and `GetSpan`\\`GetMemory` with `Advance` for writing.
+3. Never call `ToArray` and avoid calling `GetBuffer` if possible. Instead, use `GetReadOnlySequence` for reading and `GetSpan`\\`GetMemory` with `Advance` for writing. There are also miscellaneous `CopyTo` and `WriteTo` methods that may be convenient.
```

Comment:
I think it would be nice to mention that `GetSpan`\\`GetMemory` and `Advance` are from the implementation of `IBufferWriter<byte>`-

Maybe:

> `IBufferWriter<byte>.GetSpan`\\`IBufferWriter<byte>.GetMemory` with `IBufferWriter<byte>.Advance`